.gitignore

```diff
@@ -306,3 +306,4 @@ Resources/MapImages
 
 # Direnv stuff
 .direnv/
+Resources/Textures/Objects/Consumable/Drinks/thavenbadglass.rsi/master.aseprite
```

Comment:
Please do not include new .gitignore entries for your local development files.

Reply:
Removed it! sorry!!!